src/Components/Nav/index.scss

```
@@ -0,0 +1,249 @@
+%BtnSetting {
+  width: 32px;
+  height: 32px;
+  margin: 7px;
+}
+
+%BtnLogoSetting {
+  width: 18px;
+  height: 18px;
+  display: block;
+}
+
+.Nav {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 100;
+  background: #fff;
+
+  .header {
+    display: flex;
+    align-items: center;
+    max-width: 640px;
+    height: 47px;
+    margin: 0px auto;
+  }
+
+  .hamburgerBtn {
+    @extend %BtnSetting;
+    position: relative;
+
+    .hamburgerBtnLogo {
+      background: url('http://localhost:3000/images/hamburgerIcon.png') center
+        center / 18px no-repeat;
+      @extend %BtnLogoSetting;
+    }
+
+    .hamburgerBtnBadge {
+      position: absolute;
+      top: 5.45px;
+      left: 66%;
+      width: 8px;
+      height: 8px;
+      margin-left: 0px;
+      border-radius: 100%;
+      display: block;
+      background-color: rgb(255, 85, 117);
+      color: rgb(255, 255, 255);
+      border: 1px solid rgb(255, 255, 255);
+    }
+  }
+
+  .titleWrap {
+    width: 147px;
+    padding: 11px 0;
+    margin: 0px auto;
+
+    .titleLink {
+      display: block;
+      width: 147px;
+      height: 24px;
+      background: url('http://localhost:3000/images/logo.png') no-repeat;
+      background-size: 147px 24px;
+    }
+  }
+  .searchBtn {
+    @extend %BtnSetting;
+
+    .searchBtnLogo {
+      @extend %BtnLogoSetting;
+      background: url('http://localhost:3000/images/search_main.png') center
+        center / 18px no-repeat;
+    }
+  }
+
+  .cartBtn {
+    @extend %BtnSetting;
+    padding: 0;
+
+    .cartBtnLogo {
+      display: block;
+      width: 100%;
+      height: 100%;
+      background: url('http://localhost:3000/images/basketIcon.png') center
+        center / 32px no-repeat;
+    }
+  }
+}
+
+// search 모달창
+.searchModal {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: fixed;
+  top: 0px;
+  left: 0px;
+  width: 100%;
+  background: #fff;
+  z-index: 200;
+
+  .searchForm {
+    display: flex;
+    justify-content: space-around;
+    width: 640px;
+    height: 70px;
+    max-width: 640px;
+
+    .searchInputWrap {
+      position: relative;
+      margin: 14px 0px;
+      flex: 1 1 0%;
+
+      .searchInput {
+        background: url('http://localhost:3000/images/search.png') 18px center /
```

```suggestion
        background: url('/images/search.png') 18px center /
```
포트 번호가 바뀌거나 배포 되어 도메인 주소가 바뀔 경우 이미지를 찾을 수 없게 될 것 같네요. 위와 같이 수정해주세요!

---

깃 허브 이미지 업로드 후 한꺼번에 적용하겠슴니다